ci/unit/required_packages.sh

```diff
@@ -30,6 +30,7 @@ pkgs="argobots                         \
       pmix                             \
       protobuf-c                       \
       spdk-devel                       \
+      strace                           \  
```

Comment:
There is a space after `\` which messes up the linebreaks
```suggestion
      strace                           \
```

Reply:
Ty! Fixed.